TestPeerConnection_GetStats is flaky

```
--- FAIL: TestPeerConnection_GetStats (0.14s)
    stats_go_test.go:295: 
        	Error Trace:	stats_go_test.go:295
        	Error:      	"240" is not greater than or equal to "276"
        	Test:       	TestPeerConnection_GetStats
```

--- a/stats_go_test.go
+++ b/stats_go_test.go
@@ -264,6 +264,7 @@ func TestPeerConnection_GetStats(t *testing.T) {
 	})
 	assert.NoError(t, answerDC.Close())
 	waitWithTimeout(t, &dcWait)
+	time.Sleep(10 * time.Millisecond)
 
 	reportPCOffer = offerPC.GetStats()
 	reportPCAnswer = answerPC.GetStats()

